src/sentry/static/sentry/app/components/events/interfaces/spans/utils.tsx

```diff
@@ -314,3 +314,40 @@ export function generateRootSpan(trace: ParsedTraceType): SpanType {
 
   return rootSpan;
 }
+
+export function isGapSpan(span: ProcessedSpanType): span is GapSpanType {
```

Comment:
Would be simpler if we could get spans into classes and have each implementation in a subclass, but I recognize that is hard to do.